readme2tex/__main__.py

```diff
@@ -159,8 +159,15 @@
             args.rerender,
             args.bustcache)
     else:
-        # Make sure we're in a git top-level
-        assert os.path.exists(".git")
+
+        # Move ourselves to the root of the current repository
+        try:
+            root_location = check_output(['git', 'rev-parse', '--show-toplevel'])
+            os.chdir(root_location)
+        except:
+            print("The current directory is part of a git repository. Make sure that you create one before continuing")

```

Comment:
There's a slight typo here; `is` should probably be `isn't`. Let's also close off this sentence with a period.